VTUL/iawa
# LIBTD-1959: 1. Add a line to 'About' Page 2. Fix disappeared permission page

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1959) (:star:)

# What does this Pull Request do? (:star:)
This PR adds funding acknowledgement to "About" page. Also, it fixes the issue that permission page also lands on the "About" page.

# What's the changes? (:star:)

* Adds a static text line to "About" page
* Uses appropriate route for terms page

# How should this be tested?

* See if the funding acknowledgement line has been added to the end of the second paragraph
* See if permission page is displayed  when clicking on the dropdown menu from the home page

# Interested parties
Tag (@pmather ) interested parties

(:star:) Required fields
